tinygrad/tensor.py

```diff
@@ -553,13 +553,14 @@ def _broadcasted(self, fxn:Type[Function], other:Union[Tensor, float], reverse:b
   def add(self, x:Union[Tensor, float], reverse=False) -> Tensor: return self._broadcasted(mlops.Add, x, reverse) if x.__class__ is Tensor or x else self
   def sub(self, x:Union[Tensor, float], reverse=False) -> Tensor: return self._broadcasted(mlops.Sub, x, reverse) if x.__class__ is Tensor or x or reverse else self
   def mul(self, x:Union[Tensor, float], reverse=False) -> Tensor: return self._broadcasted(mlops.Mul, x, reverse) if x.__class__ is Tensor or x != 1.0 else self
-  def pow(self, x:Union[Tensor, float], reverse=False) -> Tensor:
-    if x.__class__ is not Tensor and not reverse:
+  def div(self, x:Union[Tensor, float], reverse=False) -> Tensor: return self._broadcasted(mlops.Div, x, reverse) if x.__class__ is Tensor or reverse or not x else self.mul(1/x)
```

Comment:
Why did this have to change? Was there a bug?

Reply:
div didn't change it just got moved around so that add, sub, mul, and div are right next to each other.